src/Sylius/Component/User/Model/UserInterface.php

```diff
@@ -31,6 +31,11 @@
     ToggleableInterface
 {
     const DEFAULT_ROLE = 'ROLE_USER';
+
+    /**
+     * @param string $username
+     */
+    public function setUsername($username);
```

Comment:
We have it already in Sylius, and yes we need it because none of user interface has this method.